Makefile

```diff
@@ -64,8 +64,36 @@ install-go-tools: install
 	GOBIN=$(DEPSGOBIN) go install golang.org/x/tools/cmd/goimports
 	GOBIN=$(DEPSGOBIN) go install github.com/onsi/ginkgo/ginkgo
 
+# proto compiler installation
+PROTOC_URL:=https://github.com/protocolbuffers/protobuf/releases/download/v3.15.8/protoc-3.15.8
+.PHONY: install-protoc
```

Comment:
small nit: it would be cool if we could check beforehand if the correct protoc version is already installed.

Reply:
true, plus defining the version in one place.

Reply:
handled this in latest commit.